framework/Steps.py

```diff
@@ -990,6 +991,9 @@ def _localTakeAstepRun(self,inDictionary):
         ## unpickle the ROM
         fileobj = inDictionary['Input'][i]
         unpickledObj = pickle.load(open(fileobj.getAbsFile(),'rb+'))
```

Comment:
you should remove this line if you keep the ones below

Reply:
I thought we should close this PR, it's too slow...